server/spikeCalendar.py

```diff
@@ -0,0 +1,58 @@
+from server import app
+from bs4 import BeautifulSoup
+from flask import jsonify
+import re
+import requests
+import datetime
+
+BASE_URL = "https://spike.wharton.upenn.edu/calendar/index.cfm?format=list&date="
+
+def cleanSpikeTimeString(s):
+  regex = re.compile("1?[0-9]:[0-9][0-9] [AP]M")
+  all_times = regex.findall(s)
+  return all_times
```

Comment:
All: use 4 spaces as is recommended in Python and for consistency with the rest of the project